data/beta/field/region.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: curie
+field: region
+phase: beta 
+register: ''
```

Comment:
I think this should be removed rather than empty string.

Reply:
do you mean delete row 5, or delete the empty string?

Reply:
I think delete line 5.